src/core_modules/capture-core/components/D2Form/D2Section.component.js

```diff
@@ -70,19 +66,15 @@ class D2Section extends React.PureComponent<Props> {
                     elevation={2}
                     className={classes.section}
                 >
-                    <div
-                        className={classes.sectionFieldsInSection}
-                    >
```

Comment:
removing this will remove this marging
![image](https://user-images.githubusercontent.com/4181674/100354048-66962980-2fe7-11eb-97df-1cdf056d2720.png)